crates/evm/src/executor/backend/mod.rs

```diff
@@ -609,9 +609,13 @@ impl Backend {
     /// in "failed"
     /// See <https://github.com/dapphub/ds-test/blob/9310e879db8ba3ea6d5c6489a579118fd264a3f5/src/test.sol#L66-L72>
     pub fn is_global_failure(&self, current_state: &JournaledState) -> bool {
-        let index: U256 = U256::from_str_radix(GLOBAL_FAILURE_SLOT, 16).expect("This is a bug.");
         if let Some(account) = current_state.state.get(&CHEATCODE_ADDRESS) {
-            let value = account.storage.get(&index).cloned().unwrap_or_default().present_value();
+            let value = account
+                .storage
+                .get(&Into::<U256>::into(GLOBAL_FAILURE_SLOT))
```

Comment:
`U256::from`
I saw some more of these around, this should be a clippy lint tbh
replace `Into::<(.*?)>::into` with `$1::from`

Reply:
done in https://github.com/foundry-rs/foundry/pull/5768/commits/0a68a7c45c6a0607de89a69747e94ab859e6909f